Keyboard/DefaultKeyboard.swift

```diff
@@ -39,15 +39,7 @@ func defaultKeyboard() -> Keyboard {
     keyModeChangeNumbers.toMode = 1
     defaultKeyboard.add(key: keyModeChangeNumbers, row: 3, page: 0)
     
-    if #available(iOSApplicationExtension 11.0, *) {
```

Comment:
Is this needed?

Reply:
Yes, needsInputModeSwitchKey is only available in iOS 11 and higher (the first iOS version to support iPhone X), so Xcode insists upon adding it: https://www.raywenderlich.com/139077/availability-attributes-swift